### PR TITLE
bpo-43950: include position in dis.Instruction

### PR DESCRIPTION
**Pull request by isidentical**
https://bugs.python.org/issue43950

Automerge-Triggered-By: GH:isidentical

### CODE REVIEW

**Review by pablogsal — APPROVED** [review]

File: Lib/dis.py [hunk 1]
```diff
@@ -191,8 +191,32 @@ def show_code(co, *, file=None):
     """
     print(code_info(co), file=file)
 
-_Instruction = collections.namedtuple("_Instruction",
-     "opname opcode arg argval argrepr offset starts_line is_jump_target")
+Positions = collections.namedtuple(
```
Comment (pablogsal) [inline Lib/dis.py:194]: Shoudln't this be `_Positions` the same way the other one is `_Instruction`?

File: Lib/test/test_dis.py [hunk 2]
```diff
@@ -1149,43 +1157,90 @@ def simple(): pass
...
         actual = dis.get_instructions(inner, first_line=expected_inner_line)
-        self.assertEqual(list(actual), expected_opinfo_inner)
+        self.assertInstructionsEqual(list(actual), expected_opinfo_inner)
 
     def test_jumpy(self):
         actual = dis.get_instructions(jumpy, first_line=expected_jumpy_line)
-        self.assertEqual(list(actual), expected_opinfo_jumpy)
+        self.assertInstructionsEqual(list(actual), expected_opinfo_jumpy)
+
+    def test_co_positions(self):
```
Comment (pablogsal) [inline Lib/test/test_dis.py:1201]: If the test suite is executed with the opt-out, this test is going to fail, you need to skip it if that's the case

File: Lib/dis.py (hunk 1 above)
Comment (isidentical) [reply Lib/dis.py:194]: Why? We don't need to extend the actual class with other methods (unlike in the case of `Instruction`).

Comment (pablogsal) [reply Lib/dis.py:194]: :+1:

File: Lib/test/test_dis.py (hunk 2 above)
Comment (isidentical) [reply Lib/test/test_dis.py:1201]: That is a valid concern :hmm: (do we have any more tests like this (?)). Let's introduce a decorator to `test.support`.

Comment (ammaraskar) [reply Lib/test/test_dis.py:1201]: This is an example I think https://github.com/python/cpython/blob/44f91fc802a2b71312e9abd9e8e9dcbf02e8216d/Lib/test/test_io.py#L2715-L2716

But I think the config changes should all be done together probably depending on what order we merge the PRs in.

Comment (pablogsal) [reply Lib/test/test_dis.py:1201]: Let's merge this and fix all tests in a separate (but quick) PR

Comment (isidentical) [reply Lib/test/test_dis.py:1201]: Rebasing for 0-indexed offsets...

**Pull request merged**